clinica/pipelines/atlas/compute_atlas_cli.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+import click
+
+from clinica.pipelines import cli_param
+
+pipeline_name = "compute-atlas"
+
+
+@click.command(name=pipeline_name)
+@cli_param.argument.bids_directory
+@cli_param.argument.caps_directory
+@cli_param.option.atlas_path
+def cli(
+    bids_directory: str,
+    caps_directory: str,
+    atlas_path: Optional[str] = None,
+) -> None:
+    """Cross-sectional pre-processing of T1w images with FreeSurfer.
+
+    https://aramislab.paris.inria.fr/clinica/docs/public/latest/Pipelines/T1_FreeSurfer/
+    """
```

Comment:
copy paste

Reply:
yup, and documentation is still to be done